bundles/org.openhab.binding.matter/README.md

```diff
@@ -0,0 +1,472 @@
+| doorlock-lockstate                        | Switch                   | Door Lock State              | Locks and unlocks the door and maintains the lock state                                                                                                                                                                                                              | Door             |          |             |
+| electricalpowermeasurement-activepower    | Number:Power             | Active Power                 | The active power measurement in watts                                                                                                                                                                                                                                | Energy           | true     | %.1f %unit% |
+| electricalpowermeasurement-activecurrent  | Number:ElectricCurrent   | Active Current               | The active current measurement in amperes                                                                                                                                                                                                                            | Energy           | true     | %.1f %unit% |
+| electricalpowermeasurement-voltage        | Number:ElectricPotential | Voltage                      | The voltage measurement in volts                                                                                                                                                                                                                                      | Energy           | true     | %.1f %unit% |
+
+
+## Full Example
+
+### Thing Configuration
+
+```java
+Thing configuration example for the Matter controller:
+Thing matter:controller:main [ nodeId="1" ]
+
+Thing configuration example for a Matter node:
+Thing matter:node:main:12345678901234567890 [ nodeId="12345678901234567890"]
+
+Thing configuration example for a Matter bridge endpoint:
+Thing matter:endpoint:main:12345678901234567890:2 [ endpointId=2]
+```
+
+### Item Configuration
+
+```java
+Dimmer MyDimmer "My Endpoint Dimmer" { channel="matter:node:main:12345678901234567890:1#levelcontrol-level" }
+Dimmer MyBridgedDimmer "My Bridged Dimmer" { channel="matter:endpoint:main:12345678901234567890:2#levelcontrol-level" }
+
+```
+
+### Sitemap Configuration
+
+```perl
+Optional Sitemap configuration:
+sitemap home label="Home"
+{
+    Frame label="Matter Devices"
+    {
+        Dimmer item=MyEndpointDimmer
+    }
+}
+```
+
+# Matter Bridge
+
+openHAB can also expose Items and Item groups as Matter devices to 3rd party Matter clients like Google Home, Apple Home and Amazon Alexa. This allows local control for those ecosystems and can be used instead of cloud based integrations for features like voice assistants. 
```

Comment:
```suggestion
openHAB can also expose Items and Item groups as Matter devices to 3rd party Matter clients like Google Home, Apple Home and Amazon Alexa.
This allows local control for those ecosystems and can be used instead of cloud based integrations for features like voice assistants. 
```